src/app/core/dto/date.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {Chronology} from './chronology';
+
+export interface Date {
+
+  chronology: Chronology;
+  dayOfMonth: number;
+  dayOfWeek: number;
+  dayOfYear: number;
+  hour: number;
+  minute: number;
+  month: string;
+  monthValue: number;
+  nano: number;
+  second: number;
+  year: number;
```

Comment:
Changed. But just thought provokingly.. what are the chances we will change database?

Reply:
In short term, very low. In long term, very high.

But we will need to change the returned date on the back-end anyway. I didn't change the default MongoDB structure because I didn't have enough time when I implemented it so I kept it for later.